bot.js

```diff
@@ -109,14 +109,18 @@ client.on('message', message => {
 	}
 
 	if (message.content.substring(0,5) === `${prefix}find` && isAdmin) {
-		const personToFind = message.content.split(" ")[1].toLowerCase()
-		let member = null
-		if (personToFind){
-		 member = getRoster().find(member => member.name.toLowerCase() === personToFind)
+		if (message.content.split(" ")[1] === undefined) {
+			message.channel.send(`No input, please use like this: !find <IGN>`)
+		}
+		else {
+			const personToFind = message.content.split(" ")[1].toLowerCase()
+			let member = null
+			if (personToFind){
```

Comment:
personToFind should always be defined here since we're checking that in the if statement

Reply:
add personToFind back to the top above the if and then do `if (!personToFind) (?)`